plugins/draw-tools.js

```diff
@@ -287,6 +292,9 @@ window.plugin.drawTools.manualOpt = function() {
            + '<a onclick="window.plugin.drawTools.snapToPortals();return false;" tabindex="0">Snap to portals</a>'
            + '<center><label id="MergeToggle"><input type="checkbox" '+mergeStatusCheck+' name="merge" '
            +   'onchange="window.plugin.drawTools.merge.toggle();return false;" />Reset draws before paste or import</label></center>'
+           + '<center><label id="edfToggle" style="color: #ffce00; cursor:pointer; display:block;">'
```

Comment:
leave the color/style into CSS (like the checkbox above)

Reply:
both should have a mouse pointer when on the checkbox, a finger when on the text, and the text in iitc yellow(?)
So please do a proposal.

Reply:
one does not to specify a cursor for label/input, default is enough
for the color, something like
```css
#MergeToggle, #edfToggle {
  color: thecolor
}
```

Reply:
removed style